Gemfile

```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
-gem 'bootstrap', '~> 4.0.0.alpha6'
+gem 'bootstrap', '4.0.0.alpha6'
```

Comment:
It's important to use the exact version `4.0.0.alpha6`.  If you use other versions the top nav will be problematic.